### PR TITLE
Isomorphic

### PR DESCRIPTION
**Pull request by alanclarke**
so this is kind of proof of concept, and it works, so yay, but its intended more as a discussion initiator to progress driftwood support for node

morph

### CODE REVIEW

**Comment by swagatha-christie** [comment]
By analyzing the blame information on this pull request, we identified @oliverwoodings and @roberttod to be potential reviewers

**Comment by alanclarke** [comment]
@oliverwoodings

File: src/driftwood.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+var _ = require('slapdash')
+var patterns = require('./patterns')
+var LEVELS = require('./levels')
+
+function noop () { }
+
+function createDriftwood (createConsoleLogger) {
```
Comment (alanclarke) [inline src/driftwood.js]: not a big fan of this createCreateLoggerCreatorCreator method but can't think of a better way

File: package.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "version": "1.1.3",
   "description": "A namespaced stylish logger primarily for the browser",
   "main": "index.js",
```
Comment (oliverwoodings) [inline package.json]: I thought the main entry point was going to stay as the browser? Or does SSG support the browser field?

Comment (alanclarke) [inline package.json]: SSG supports the browser field, which replaces main with browser value when in browser

Comment (oliverwoodings) [inline package.json]: Awesome!

File: src/__tests__/browserLogger.js [hunk 3]
```diff
@@ -1,32 +1,25 @@
 var expect = require('../../test/expect')
```
Comment (oliverwoodings) [inline src/__tests__/browserLogger.js]: Should this file be called browserLoggerTests?

Comment (alanclarke) [inline src/__tests__/browserLogger.js]: yes, soz!

File: src/loggers/node.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+var levelColors = {
+  trace: 'gray',
+  debug: 'green',
+  info: 'blue',
+  warn: 'orange',
+  error: 'red'
+}
+
+module.exports = function log (name, level, message, meatdata) {
```
Comment (oliverwoodings) [inline src/loggers/node.js]: image

**Comment by oliverwoodings** [comment]
This looks fantastic!

Before we deploy, would you be able to test the new version with smartserve? Want to make sure the browser field stuff works ok.

**Comment by alanclarke** [comment]
Ah cool, thanks! yeah still needs some more work and will do some thorough testing

**Comment by oliverwoodings** [comment]
@alanclarke what happened to this?

**Comment by oliverwoodings** [comment]
Todo:

- [x] Add timestamps to node output
- [ ] Make persisting optional and default to false
- [ ] Update tests to test persistence
- [ ] Fix CI

**Comment by oliverwoodings** [comment]
Closes #35, closes #34, closes #33, closes #32, closes #21, closes #13, closes #12, closes #11, closes #10

**Comment by oliverwoodings** [comment]
screen shot 2016-09-18 at 17 57 47

**Comment by oliverwoodings** [comment]
Plan is to roll this out once everyone is on the webpack branch since webpack will respect the browser field.

**Pull request merged**